iocore/net/SSLClientUtils.cc

```diff
@@ -66,6 +66,7 @@ verify_callback(int preverify_ok, X509_STORE_CTX *ctx)
   netvc = SSLNetVCAccess(ssl);
 
   if (netvc != nullptr) {
+    netvc->callHooks(TS_EVENT_SSL_SERVER_VERIFY_HOOK);
```

Comment:
For the simple cases where we want to abort / continue verification wouldn't simply reenabling with an error/continue do the trick?
In order to support skipping remaining core verification we could add a flag (+API to set it). Alternatively, we could make proxy.config.ssl.client.verify.server overridable and check that.